cms/djangoapps/contentstore/views/videos.py

```diff
@@ -297,6 +297,7 @@ def videos_post(course, request):
 
     The returned array corresponds exactly to the input array.
     """
+    supported_content_types = ['video/mp4', 'video/quicktime']
```

Comment:
There are so many non-video files. I think, the best is we check for supported file formats.